src/sql/parts/jobManagement/views/jobStepsView.component.ts

```diff
@@ -17,14 +17,15 @@ import { DashboardServiceInterface } from 'sql/parts/dashboard/services/dashboar
 import { AgentJobHistoryInfo } from 'sqlops';
 import { JobStepsViewController, JobStepsViewDataSource, JobStepsViewFilter,
 	JobStepsViewRenderer, JobStepsViewRow, JobStepsViewModel} from 'sql/parts/jobManagement/views/jobStepsViewTree';
+import { JobHistoryComponent } from './jobHistory.component';
```

Comment:
do we have any coding conventions for using relative vs. full import paths?  I typically use the full path, which seems like the case for all the other imports in this file.